CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Add `autocomplete` attribute option to input component
+* Use `autocomplete=email` on feedback component's survey signup form
```

Comment:
This should be a list of the PRs, not a list of the changes. So this entry should be:

`- Add autocomplete attribute to feedback form (PR #733)`